__tests__/integration.test.js

```diff
@@ -33,36 +33,81 @@ describe("/api/topics", () => {
+            });
 
-    })
-    test('ERROR: returns 404 error if there is an invalid point entered.',()=>{
+    });
+    test('ERROR: returns 404 error if there is an invalid point entered.', () => {
         return request(app)
-        .get("/api/tApics")
-        .expect(404)
-        .then(({body})=>{
-            const {message} = body;
-            expect(message).toBe("Error 404! endpoint not found :(")
-        });
+            .get("/api/tApics")
+            .expect(404)
+            .then(({ body }) => {
+                const { message } = body;
+                expect(message).toBe("Error 404! endpoint not found :(")
+            });
     });
 
-})
+});
 
-describe('Being on api',()=>{
-    test("Responds with an object describing all the aviable endpoints in this API",()=>{
+describe('GET:/api', () => {
+    test("Responds with an object describing all the aviable endpoints in this API", () => {
         return request(app)
-        .get("/api")
-        .expect(200)
-        .then(({body})=>{
-            const {endpoint} = body
-            expect(typeof endpoint).toBe("object")
+            .get("/api")
+            .expect(200)
+            .then(({ body }) => {
+                const { endpoint } = body
+                expect(typeof endpoint).toBe("object")
+                expect(endpoint.hasOwnProperty("GET /api")).toBe(true)
+                expect(endpoint.hasOwnProperty("GET /api/topics")).toBe(true)
+                expect(endpoint.hasOwnProperty("GET /api/articles")).toBe(true)
+                expect(endpoint.hasOwnProperty("GET /api/articles/:article_id")).toBe(true)
+            });
+    });
+
+});
+
+describe("/api/articles", () => {
+    test("GET:/api/articles/article:id returns the specified article", () => {
+        return request(app)
+            .get('/api/articles/6')
+            .expect(200)
+            .then(({ body }) => {
+                const { article } = body
+                expect(typeof article).toBe("object")
+                expect(article.hasOwnProperty('article_id')).toBe(true)
+                expect(article.hasOwnProperty('article_img_url')).toBe(true)
+                expect(article.hasOwnProperty('author')).toBe(true)
+                expect(article.hasOwnProperty('body')).toBe(true)
+                expect(article.hasOwnProperty('created_at')).toBe(true)
+                expect(article.hasOwnProperty('title')).toBe(true)
+                expect(article.hasOwnProperty('topic')).toBe(true)
+                expect(article.hasOwnProperty('votes')).toBe(true)
+            })
+
+    })
+
+    test("GET: Returns an error message when provided an invalid article id",()=>{
+        return request(app)
+        .get('/api/articles/papa_pear')
+        .expect(400)
+        .then(({ body }) => {
+            const { message } = body;
+            expect(message).toBe("Bad request :(")
+        })
+    })
+
+    test.only("GET: Returns an error message when provided an higher id number",()=>{
```

Comment:
remove .only from your tests before pushing otherwise we can't be sure that all of your tests pass!